tests/test_schema.py

```diff
@@ -211,8 +211,8 @@ class Model(BaseModel):
 
 def test_schema_repr():
     s = Field(4, title='Foo is Great')
-    assert str(s) == "default=4 title='Foo is Great' extra={}"
-    assert repr(s) == "FieldInfo(default=4, title='Foo is Great', extra={})"
+    assert str(s) == "default=4 title='Foo is Great' allow_mutation=True extra={}"
```

Comment:
I guess we should also overwrite `__repr_args__` as we probably don't want this to be set by default

Reply:
To do so, we'll have to have access to the defaults within the `Representation` class.  This ties in with your previous comment to set the defaults once and make them available everywhere.  I don't think I'm familiar enough with the code to make this decision, but I'm happy to make the changes in this PR if I'm given guidance.  I think updating `__slots__` to include the defaults could possibly work... but several classes extend `Representation`, so not sure if that's the correct approach.

Reply:
After thinking about it some more, I found an approach I liked and went ahead and implemented it.  I simply pushed the get_constraints function from `schema.py` to a method on `FieldInfo`.  The rest came naturally after that.